web/styles/informational_overlays.css

```diff
@@ -1,108 +1,186 @@
 .informational-overlays {
+    display: grid;
+    place-items: center;
+
     .overlay-content {
-        /* because zoom breaks at 525px perhaps due to rounding errors, so add a
-           trivial amount of width so it doesn't break. */
-        width: 550px;
+        width: 90%;
+        max-width: 650px;
         margin: 0 auto;
         position: relative;
-        top: calc((30vh - 50px) / 2);
-        border-radius: 4px;
+        top: unset;
```

Comment:
Again, because you're using Grid, you can strike a whole bunch of these properties entirely. `position: relative` doesn't lijkely need to happen, now that `.overlay-content` is a grid item. That means you don't even need a `top` declaration. `margin: 0 auto` can disappear to.

One thing I like to do when I'm working with CSS is to use my browsers inspector and just switch off different CSS properties. If there's no effect from switching the property off, I'll just delete it in my source code, and then spend some time browsing around to see if any distant elements are affected.

For example:

| Original | Switched off |
| --- | --- |
| <img width="995" alt="initial-values" src="https://github.com/zulip/zulip/assets/170719/fe7f7783-cbde-455d-9aa3-97929f1670bb"> | <img width="995" alt="switched-off" src="https://github.com/zulip/zulip/assets/170719/d67d5d0e-3ec5-42b9-9753-cf77622e4bba"> |

The overlay seems unaffected by those properties being switched off. So I'd delete them.